src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
Random files in:
Linux kernel: http://lxr.free-electrons.com/source/drivers/virtio/virtio_pci_modern.c
iptables: http://git.netfilter.org/iptables/tree/iptables/ip6tables.c
QEMU: http://git.qemu.org/?p=qemu.git;a=blob;f=hw/pci-host/ppce500.c;h=50add34e509f4e8b6e110e5166e0dcf7bac2de12;hb=HEAD
OpenStack neutron: https://github.com/openstack/neutron/blob/master/neutron/plugins/ml2/drivers/mech_sriov/mech_driver/mech_driver.py

shows that this is not that uncommon practice to have the licensing and copyright info in the file. It is actually the opposite - authors release their work under certain license while they keep the copyright. And this is clearly stated in the file itself, not in some external place.
I agree that `git` can do its job very good, but this won't work once you start packaging where usually they work with exported archives of the sources.
And I certainly don't agree that the license and copyright are "random notes". There are lot's of people that make big money out of such things, so ...